taskcluster/tc-brew-tests.sh

```diff
@@ -24,15 +24,14 @@ install_local_homebrew()
 
     for suffix in .brew .cache .logs;
     do
-        if [ -d "${TASKCLUSTER_TASK_DIR}/${LOCAL_HOMEBREW_PREFIX}${suffix}/" ]; then
-            echo "Directory ${TASKCLUSTER_TASK_DIR}/${LOCAL_HOMEBREW_PREFIX}${suffix} already exists, aborting"
-            exit 1
+        if [ -d "${TASKCLUSTER_TASK_DIR}/homebrew.local/${LOCAL_HOMEBREW_PREFIX}${suffix}/" ]; then
+            echo "WARNING: Directory ${TASKCLUSTER_TASK_DIR}/homebrew.local/${LOCAL_HOMEBREW_PREFIX}${suffix} already exists"
```

Comment:
Is this warning when we encounter the expected behavior? Why?

Reply:
That was a way to ensure finding that easily in the logs